### DISCUSSION TITLE
ActionIcon - Dumi Theme LobeHub

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#34] [Q&A]
# ActionIcon - Dumi Theme LobeHub

ActionIcon is a component used to render an icon with a block around it

http://localhost:8000/components/action-icon

**Comment by huxinfeng** [comment]
哈喽

**Reply by jinfei-lab** [reply]
哈喽

**Comment by canisminor1990** [comment]
👀
